eZ/Bundle/EzPublishRestBundle/Resources/config/routing.yml

```diff
@@ -773,7 +773,7 @@ ezpublish_rest_loadRoleDraft:
 ezpublish_rest_loadRoleDraftByRoleId:
     path: /user/roles/{roleId}/draftByRoleId
```

Comment:
@bdunogier We've got both ezpublish_rest_loadRoleDraft, path: /user/roles/{roleId}/draft
and ezpublish_rest_loadRoleDraftByRoleId, path: /user/roles/{roleId}/draftByRoleId
...because roles and role drafts have different IDs. However, both use the same underlying controller method, which first tries to load a draft by role ID, and if that fails it tries by role draft ID. Given all that, it seems me may only need the first route.